NEWS

```diff
@@ -2,6 +2,26 @@ systemd System and Service Manager
 
 CHANGES WITH 240 in spe:
 
+        * NoNewPrivileges=yes has been set for all long-running services
+          implemented by systemd. Previously, this was problematic due to
+          SELinux (as this would also prohibit the transition from PID1's label
+          to the service's label). This restriction has since been lifted, but
+          a SELinux POLICY UPDATE is required.
```

Comment:
hmm, I'd pronounce SELinux "aass eeee linux", and hence would assume a → an?

Also, why uppercase POLICY UPDATE? we usllay try to avoid yelling at people ;-)